documentation/modules/security/ref-certificates-and-secrets.adoc

```diff
@@ -162,6 +162,36 @@ NOTE: `_<cluster>_-clients-ca` is used to sign certificates of client applicatio
 It needs to be accessible to the Strimzi components and for administrative access if you are intending to issue application certificates without using the User Operator.
 You can enforce this using Kubernetes role-based access controls if necessary.
 
+== Adding labels and annotations to Secrets
+
+By configuring the `clusterCaCert` template property, you can add custom labels and annotations to the Cluster CA Secrets created by the Cluster Operator.
```

Comment:
_Suggested change:_

By configuring the `clusterCaCert` template property > By configuring the `clusterCaCert` template property in the `Kafka` custom resource